charts/karmada-operator/values.yaml

```diff
@@ -0,0 +1,66 @@
+## Default values for charts.
+## This is a YAML-formatted file.
+## Declare variables to be passed into your templates.
+
+## @param global karmada global config
+global:
+  ## @param global.imageRegistry Global Docker image registry
+  imageRegistry: ""
+  ## E.g.
+  ## imagePullSecrets:
+  ##   - myRegistryKeySecretName
+  imagePullSecrets: []
+
+## operator manager config
+operator:
+  ## @param operator.labels
+  labels: {}
+  ## @param operator.replicaCount target replicas
+  replicaCount: 1
+  ## @param operator.podAnnotations
+  podAnnotations: {}
+  ## @param operator.podLabels
+  podLabels: {}
+  ## @param image.registry operator image registry
+  ## @param image.repository operator image repository
+  ## @param image.tag operator image tag (immutable tags are recommended)
+  ## @param image.pullPolicy operator image pull policy
+  ## @param image.pullSecrets Specify docker-registry secret names as an array
+  ##
+  image:
+    registry: docker.io
+    repository: karmada/karmada-operator
+    tag: latest
```

Comment:
I think use latest tag is not a good idea.

Reply:
thanks for your review. 
Now, we only have the latest version. 

the pkg `charts/karmada-operator/charts/common-1.17.1.tgz` is not need to commit indeed.

Reply:
> CI what you say is chart lint?

Yeah, samething of karmada helm chart. ref:
- https://github.com/karmada-io/karmada/blob/master/.github/workflows/lint-chart.yaml

Reply:
I think we can hold a while for it. We can improve it separately in the future.